### DISCUSSION TITLE
Plugin to list albums by an artist NOT in the user's library?

### DISCUSSION THREAD

**Discussion by tomauty** [beetbox/beets#5532] [Q&A]
Hi all,

Does anyone know if there currently exists a plugin or simple way to search for albums in your library you _don't_ have (on the basis of artist, maybe label, maybe it's customizable)? I would love to set this up to do make an initial file of albums I might want to obtain, and then periodically run it again as a sort of 'shopping list' or new release tracker. If not, I'll probably start making one for personal use and if there's interest -- put it out there somewhere.

Thanks!

**Comment by mkmtrading** [comment]
I'm a newbie but I played around with the plug-in missing and it lists all the albums that I don't have (probably because I've misconfigured it)

**Reply by tomauty** [reply]
Interesting, I've used `missing` a bunch but have never noticed this:
>  It can also list the names of albums that your library is missing from each artist.

Let me mess with the `-a` arg and see if that gives what I'm looking for. Ty!
